### DISCUSSION TITLE
Source of the icons used in the application?

### DISCUSSION THREAD

**Discussion by benjysugi** [sqlitebrowser/sqlitebrowser#2961] [Q&A]
does anyone know the source of the icons used in the application / where they came from?
i've seen them used in various other apps but i want to know what their origins are
![link to example](https://cdn.discordapp.com/attachments/818950958246789141/937738894340919336/unknown.png)

**Comment by chrisjlocke** [comment]
They're part of the 'silk icons' set by 'Fam fam fam'.
http://www.famfamfam.com/lab/icons/silk/

There is also a set by 'Damien' who's created another 450 in the same style.
https://damieng.com/blog/2007/11/19/more-silk-icons-silk-companion-1/

I use them regularly in my own applications.  Lovely icons.
